src/bin/tikv-server.rs

```diff
@@ -71,8 +71,8 @@ fn get_string_value<F>(short: &str,
 fn initial_log(matches: &Matches, config: &toml::Value) {
     let level = get_string_value("L",
                                  "server.log-level",
-                                 &matches,
-                                 &config,
+                                 matches,
```

Comment:
as we updated rust and clippy, compile output has many warning...
this is a small change and don't the affect logic